### DISCUSSION TITLE
DevEX feedback: Possible enhancements.

### DISCUSSION THREAD

**Discussion by MetzinAround** [projectdiscovery/nuclei#4326] [Ideas]
I have a friend who's a principal engineer at UBER, and he played with nuclei for a bit. This is a list of feedback he gave me. 

"Gave a test run of Nuclei in docker, here's a bit of devx feedback from spending a few minutes on it.

● The difference in PDTM and underlying tools isn't immediately clear. The way the site organizes into several other domains and their own docs isn't super clear to follow for new users.
● There's quite an opinionated choice to expose the go package manager to the end-user for installation. Even resulting in more of the blog about isntalling Go than using the tools. This should probably be minimized and just one way to get the package.
● Should distribute a .deb, especially if you want to target Kali linux or security distros based on debian.
● Your base container for nuclei is pretty large, should probably use Alpine.
● PDTM should probably have a container distributed as well that can manage and configure all the underlying tools in one container.
● Nuclei should have an optional for an html output as well."

I'm not sure what's valuable here, or what you already have considered/decided against. But this is something I figured was good feedback to introduce.

**Comment by MetzinAround** [comment]
He's very busy right now, but hopefully in the future I can chat more with him about how he used it, what he likes, etc.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
